### PR TITLE
- Send to contact fixes(Remaining batch transaction flow)

### PR DESCRIPTION
**Pull request by cakesoft-nikhita**

### CODE REVIEW

File: src/common/data/models/interfaces/RecipientDescribing.ts [hunk 1]
```diff
@@ -62,15 +62,15 @@ export function makeContactRecipientDescription(
   return {
     id: data.id,
     kind: RecipientKind.CONTACT,
-    displayedName: data.contactName,
-    walletName: data.contactsWalletName,
-    avatarImageSource: null,
+    displayedName: data.contactName || data.displayedName,
```
Comment (AliMeer) [inline src/common/data/models/interfaces/RecipientDescribing.ts:65]: Why do we need multiple variable names for `displayName` ?

Can we not have jus` data.displayName?`

File: src/common/data/models/interfaces/RecipientDescribing.ts [hunk 2]
```diff
@@ -62,15 +62,15 @@ export function makeContactRecipientDescription(
...
+    walletName: data.contactsWalletName || data.walletName,
```
Comment (AliMeer) [inline src/common/data/models/interfaces/RecipientDescribing.ts:66]: Same applies for walletName... lets see if we can normalise this in the next PR.

**Review by AliMeer — APPROVED** [review]

File: src/common/data/models/interfaces/RecipientDescribing.ts (hunk 1 above)
Comment (cakesoft-nikhita) [reply src/common/data/models/interfaces/RecipientDescribing.ts:65]: Actually, this object is referring two times when the user picks data from System contact then the key name is "contactName" and when on SendToContact flow again this object is referring then we have displayName

File: src/common/data/models/interfaces/RecipientDescribing.ts (hunk 2 above)
Comment (cakesoft-nikhita) [reply src/common/data/models/interfaces/RecipientDescribing.ts:66]: Same as above this object is referring to 2 flows so, I need to handle this

**Pull request merged**